scripts/test

```diff
@@ -11,6 +11,6 @@ bosh upload-release https://bosh.io/d/github.com/cloudfoundry/syslog-release
 bosh upload-release https://bosh.io/d/github.com/cloudfoundry-incubator/windows-utilities-release
 
 pushd "$(dirname "$0")/../tests"
-  go install github.com/onsi/ginkgo/ginkgo@latest
+  go install github.com/onsi/ginkgo/v2/ginkgo@latest
   ginkgo -r -nodes=2 "$@"
```

Comment:
The ginkgo v2 CLI prefers `--procs=2`, as `-nodes` is deprecated.

Reply:
Replaced -nodes with --procs, thank you!